andela/AndelaEatsUI
# fix/ implemented loader

**What does this PR do?**
Fix Loader for the order page

**Description of Task to be completed?**
added a loading state

**How should this be manually tested?**
when the application is started using `npm run dev` you will see a loader displayed before the meal options returns

**What are the relevant pivotal tracker stories?**
159230197

**Any background context you want to add?**
N/A

**Screenshots**
